lib/bamboo/email.ex

```diff
@@ -1,8 +1,8 @@
 defmodule Bamboo.Email do
   defstruct from: nil,
-      to: nil,
-      cc: nil,
-      bcc: nil,
+      to: [],
+      cc: [],
+      bcc: [],
```

Comment:
Actually I think this should go back to nil and it should raise it all recipients are nil. That would mean you forgot to set something